### PR TITLE
Updated main README and added API docs to Node app

### PR DESCRIPTION
**Pull request by Fleeck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,34 +4,34 @@ This repository contains sample projects and starter templates for building Apps
...
+| Calling Tradeshift API | X | X | X 
+| Front-end API integration  | X | X | X 
+| Using [Tradeshift's UI Components](http://ui.tradeshift.com), including table and tabs  | X | X | X 
 | i18n support on the client side  | X | X | X 
 | i18n support on the server side  | X | X | X 
 | Check health function for monitoring  | X | X | X 
-| Unit test framework to get you started  | X | X | X 
+| Front-end basic test to get you started  | X | X | X 
+| Back-end basic test to get you started  | X | X | 
+| Documented local development using [ngrok](https://ngrok.com/) |  | X |
```
Comment (jbarmash) [inline README.md]: this should not be in the table.  Please move ngrok text to the main readme.

File: README.md [hunk 2]
```diff
@@ -4,34 +4,34 @@ This repository contains sample projects and starter templates for building Apps
...
-|OAUTH2 Connection | X | X | X 
-|Authorization with Tradeshift Servers using OAUTH2 three-legged approach  | X | X | X 
-| Tradeshift APIs - getting documents  | X | X | X 
-| Tradeshift APIs - user information.  | X | X | X 
-| Displaying CompanyCard and Locale  | X | X | X
-| Angular js frontend talking to RESTful backend | X | X | X 
-| [Tradeshift's UI Components](http://ui.tradeshift.com), including table and tabs  | X | X | X 
+| OAUTH2 Connection | X | X | X 
+| Authorization with Tradeshift Servers using OAUTH2 three-legged approach ([docs](http://apps.tradeshift.com/developers/documentation/#app-authentication))  | X | X | X 
+| Calling Tradeshift API | X | X | X
```
Comment (jbarmash) [inline README.md]: "Calling Tradeshift API" is less detail than previously.  Showing how to call different specific APIs is better.  Please restore more detail.

File: README.md [hunk 3]
```diff
@@ -4,34 +4,34 @@ This repository contains sample projects and starter templates for building Apps
...
-|Authorization with Tradeshift Servers using OAUTH2 three-legged approach  | X | X | X 
-| Tradeshift APIs - getting documents  | X | X | X 
-| Tradeshift APIs - user information.  | X | X | X 
-| Displaying CompanyCard and Locale  | X | X | X
-| Angular js frontend talking to RESTful backend | X | X | X 
-| [Tradeshift's UI Components](http://ui.tradeshift.com), including table and tabs  | X | X | X 
+| OAUTH2 Connection | X | X | X 
+| Authorization with Tradeshift Servers using OAUTH2 three-legged approach ([docs](http://apps.tradeshift.com/developers/documentation/#app-authentication))  | X | X | X 
+| Calling Tradeshift API | X | X | X 
+| Front-end API integration  | X | X | X
```
Comment (jbarmash) [inline README.md]: Please restore "Angular.js frontend talking to RESTful backend "

File: README.md [hunk 4]
```diff
@@ -4,34 +4,34 @@ This repository contains sample projects and starter templates for building Apps
...
-| Unit test framework to get you started  | X | X | X 
+| Front-end basic test to get you started  | X | X | X 
+| Back-end basic test to get you started  | X | X | 
+| Documented local development using [ngrok](https://ngrok.com/) |  | X |   
 | Tasks Service Example  | X |  |  
 | Webhooks Example  | X |  |  
 
 
-The sample apps all aim to show same things, and the front end code talking to different front ends is almost identical: 
+The sample apps all aim to show same things, and the Front-end code talking to different Back-ends is almost identical:
```
Comment (jbarmash) [inline README.md]: Is it almost identical?  it should be identical.  What are the differences?

File: README.md (hunk 1 above)
Comment (Fleeck) [inline README.md]: To the main readme? It is specified in apps. DO we need to duplicate it?

File: README.md (hunk 2 above)
Comment (Fleeck) [inline README.md]: Changed

File: README.md (hunk 3 above)
Comment (Fleeck) [inline README.md]: Changed

File: README.md (hunk 4 above)
Comment (Fleeck) [inline README.md]: Front-end code is not talking to different front-ends, but to back-ends :)

**Comment by jbarmash** [comment]
👍 LGTM.

**Pull request merged**